### ISSUE TITLE
When will release the code of collecting dataset?

### ISSUE THREAD

**Issue by zjwzcx** [XiaohanLei/GaussNav#4]
Thanks for your awesome works. I'm curious about the implementation details of frontier-based exploration. When will release the code of collecting dataset?

**Comment by HannahHaensen** [comment]
+1

**Comment by luolina116-cmd** [comment]
Hello, have you written the code for collecting the data? I'm encountering some difficulties in generating this dataset (the data used to generate the semantic Gaussian map).

**Comment by zjwzcx** [comment]
> Hello, have you written the code for collecting the data? I'm encountering some difficulties in generating this dataset (the data used to generate the semantic Gaussian map).

No... We finally use Gibson/MP3D/ProcTHOR/HSSD meshes in our work https://github.com/zjwzcx/GLEAM.
